### DISCUSSION TITLE
Discover TF of gene X.

### DISCUSSION THREAD

**Discussion by MohammadIzadi** [aertslab/scenicplus#217] [Q&A]
Hi,

Thank you for developing SCENIC+. I am excited to use it.
I have two questions:
first, Can I find the Transcription Factor (TF) of the specific gene (X gene) by SCENIC+. for example, can I cluster and label cells in sc-RNA seq data by expression of gene X and then use SCENIC+ to find TF of gene X? For this purpose, Should I label cells of scATAC sequencing data by peak of downstream or upstream of gene X?

and the second one is I have non multiome data. I have one sample and I have tested it separately. first, I did scRNA-seq for this sample and then again, I did scATAC-seq. and the cell number of my scATAC-seq data was more than scRNA-seq data. Can I use these two data for analysis by SCENIC+? How can I do this?

Sincerely,
Izadi

**Comment by SeppeDeWinter** [comment]
Hi

Regarding your first question, SCENIC+ is an unsupervised method, for this reason it's not possible to _only_ find the TFs regulating a single gene. However, you can still run SCENIC+. It will find multiple TFs for multiple genes. Afterwards you can have a look which TFs regulate your particular gene of interest.

Regarding your second questions, It is still possible to run SCENIC+ as long as you are able to match the cells (or clusters) in your scATAC-seq data to the cells (or clusters) in your scRNA-seq data. See this tutorial: https://scenicplus.readthedocs.io/en/latest/mix_melanoma_cell_lines.html.

Good luck with your analysis and thank you for using SCENIC+.


All the best,

Seppe
